src/axom/inlet/Proxy.hpp

```diff
@@ -201,9 +218,9 @@ class Proxy
   }
 
 private:
-  Table* m_table = nullptr;
-  Field* m_field = nullptr;
-  Function* m_func = nullptr;
+  const Table* const m_table = nullptr;
+  const Field* const m_field = nullptr;
+  const Function* const m_func = nullptr;
```

Comment:
Re: printing the path -- looks like it doesn't  store any additional information.